Ellen_input.py

```diff
@@ -0,0 +1,11 @@
+#TO DO: create a valid input function
+
+def valid_input():
+    while True:
+        response = input('Is Arcade the best programmer ever!? Type Y or N. \n')
+        if response.lower() not in ('y', 'n', 'yes', 'no'):
+            print('Sorry, that is not a valid answer. Please try again and enter a valid input. Type either Y or N. \n')
+        else:
+            break
+
```

Comment:
Love that you used a while loop, one adjustment I'd make is passing the options you want into the function to use, but we can go over that today :)